src/components/language/dto/language.dto.ts

```diff
@@ -155,6 +155,9 @@ export class Language extends Resource {
   @Field()
   readonly hasExternalFirstScripture: SecuredBoolean;
```

Comment:
is this still necessary? would the `ExternalFirstScripture` in the first-scripture.dto cover for this?